.eslintrc

```diff
@@ -3,14 +3,26 @@
     "es6": true,
     "browser": true
   },
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": 2020,
+    "sourceType": "module",
+    "project": "./tsconfig.eslint.json"
+},
   "globals": {
     "Thenable": "readonly",
     "VSCODE_VERSION": "readonly"
   },
   "extends": [
     "@codingame"
   ],
-  "ignorePatterns": "vscode/**/*",
-  "plugins": ["@typescript-eslint", "import", "unused-imports"]
-
+  "ignorePatterns": [
+    "vscode/**/*",
+    "dist/**/*"
+  ],
+  "plugins": [
+    "@typescript-eslint",
+    "import",
+    "unused-imports"
+  ]
```

Comment:
I'm confused, I was thinking this project uses `@codingame/eslint-config`. It's installed but not used...

Reply:
And I just checked, I didn't change/remove it by accident

Reply:
Nop indeed, it's probably my fault!

Reply:
Ok, I guess, you will address this independent of this PR.